src/diffusers/models/attention.py

```diff
@@ -1087,8 +1104,15 @@ def forward(
                 accumulated_values[:, frame_start:frame_end] += hidden_states_chunk * weights
                 num_times_accumulated[:, frame_start:frame_end] += weights
 
-        hidden_states = torch.where(
-            num_times_accumulated > 0, accumulated_values / num_times_accumulated, accumulated_values
+        hidden_states = torch.cat(
```

Comment:
Better to include this change in the memory optimisations no?

Reply:
Sounds good! Will revert here